docs/CaliptraIntegrationSpecification.md

```diff
@@ -58,16 +58,16 @@ The following table describes integration parameters.
 
 *Table 2: Integration parameters*
 
-| **Parameter name** | **Width** | **Description** |
-| :--------- | :--------- | :--------- |
-| APB_ADDR_WIDTH | 32 | Width of the APB Address field. Default to 32. |
-| APB_DATA_WIDTH | 32 | Width of the APB Data field. Default to 32. |
-| APB_USER_REQ_WIDTH | TODO | Width of the APB PAUSER field. |
-| CPTRA_SET_MBOX_PAUSER_INTEG | 5 | Each bit hardcodes the valid PAUSER for mailbox at integration time. |
-| CPTRA_MBOX_VALID_PAUSER | \[4:0\]\[31:0\] | Each parameter corresponds to a hardcoded valid PAUSER value for mailbox, set at integration time. Must set corresponding bit in the CPTRA_SET_MBOX_PAUSER_INTEG parameter for this valid pauser override to be used. |
-| CPTRA_DEF_MBOX_VALID_PAUSER | 32 | Sets the default valid PAUSER for mailbox accesses. This PAUSER is valid when any VALID_PAUSER is unlocked or not set by INTEG parameter. |
-| CPTRA_SET_FUSE_PAUSER_INTEG | 1 | Sets the valid PAUSER for fuse accesses at integration time. |
-| CPTRA_FUSE_VALID_PAUSER | 32 | Overrides the programmable valid PAUSER for fuse accesses when CPTRA_SET_FUSE_PAUSER_INTEG is set to 1. |
+| **Parameter name** | **Width** | **Defines File** | **Description** |
+| :--------- | :--------- | :--------- | :--------- |
+| CALIPTRA_APB_ADDR_WIDTH | 32 | config_defines.svh | Width of the APB Address field. Default to 32. |
```

Comment:
Since these are defines, not parameters, they should be listed below in Table 3

Reply:
Done!